source/fe/mapping_q_generic.cc

```diff
@@ -3621,82 +3623,70 @@ namespace
    * Boundary<dim,spacedim>.
    */
   template <int dim, int spacedim, class TriaIterator>
-  void get_intermediate_points_on_object(const Manifold<dim, spacedim> &manifold,
-                                         const QGaussLobatto<1>        &line_support_points,
-                                         const TriaIterator &iter,
-                                         std::vector<Point<spacedim> > &points)
+  void add_intermediate_points(const Manifold<dim, spacedim> &manifold,
+                               const Table<2,double>         &weight_table,
+                               const TriaIterator            &iter,
```

Comment:
Also while you're there: call the template argument `TriaIteratorType`.

One could probably avoid the first two template arguments of the function by getting the information from the iterator object. But that's something we can do at a later time.